### PR TITLE
Docs Update: OpenWayback -> pywb Transition Guide

### PR DESCRIPTION
**Pull request by ikreymer**
## Description

This PR adds the OpenWayback to pywb transition guide, which will be available on https://pywb.readthedocs.io/
This also includes sample deployment configurations in `sample-deploy`, which include:
- a sample deployment with OutbackCDX
- a sample for how to deploy with Nginx on a subdirectory
- a sample for how to deploy with Apache on a subdirecory

Also: bumping to latest wombat, bump version to 2.4.3

### CODE REVIEW

File: docs/manual/owb-to-pywb-deploy.rst [hunk 1]
```diff
@@ -0,0 +1,65 @@
+Deploying pywb: Collection Paths and with Nginx/Apache
+======================================================
+
+In pywb, the collection name is also the access point and each the collections in ``config.yaml``
```
Comment (ldko) [inline docs/manual/owb-to-pywb-deploy.rst:4]:
```suggestion
In pywb, the collection name is also the access point, and each of the collections in ``config.yaml``
```

File: docs/manual/owb-to-pywb-deploy.rst [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+      collections:
+        $root:
+            ...
+
+        another-collection:
+            ...
+
+
+Now, the first collection from: ``http://<hostname>/*/https://example.com/``
```
Comment (ldko) [inline docs/manual/owb-to-pywb-deploy.rst:32]:
```suggestion
Now, the first collection is available at: ``http://<hostname>/*/https://example.com/``.
```

File: docs/manual/owb-to-pywb-deploy.rst [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+See the :ref:`nginx-deploy` and :ref:`apache-deploy` sections for more info on deploying with Nginx and Apache.
+
+
+Working Docker Compose Examples
+-------------------------------
+
+The pywb `Deployment Examples <https://github.com/webrecorder/pywb/blob/docs/sample-deploy/>`_ include working examples of deploying pywb with Nginx, Apache and OutbackCDX
+in Docker using Docker Compose.
+
+The examples are availablee in the ``sample-deploy`` directory of the pywb repo. The examples include:
```
Comment (ldko) [inline docs/manual/owb-to-pywb-deploy.rst:57]:
```suggestion
The examples are available in the ``sample-deploy`` directory of the pywb repo. The examples include:
```

File: docs/manual/outbackcdx.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+.. code:: console
+
+    cdx-indexer /path/to/mywarcs/my.warc.gz > ./index.cdx
+    curl -X POST --data-binary @index.cdx http://localhost:8084/mycoll
+
+The contents of the index are added to the ``mycoll`` index, which can correspond to the web archive collection ``mycoll``.
+The index is created automatically if it does not exist.
+
+The above can be repeated for each WARC file, or for the entire collection if small (by using a wildcard, eg. ``cdx-indexer /path/to/mywarcs/*.warc.gz > ./index.cdx``). If the CDX index is too big, OutbackCDX may fail and ingesting an index per-WARC is recommended.
+
```
Comment (ldko) [inline docs/manual/outbackcdx.rst:31]: Based on the OutbackCDX docs it seems like if you have cdx files you were already using with OpenWayback, that data can be POSTed to OutbackCDX without reindexing with the pywb `cdx-indexer`. Is that an accurate or is it recommended to reindex the [W]ARCs anyway?

**Review by ldko — COMMENTED** [review]
This review is mostly typo fix suggestions but also a couple questions that came up while reading over the migration guide. Thanks, Ilya!

File: docs/manual/owb-to-pywb-deploy.rst [hunk 5]
```diff
@@ -0,0 +1,65 @@
+Deploying pywb: Collection Paths and with Nginx/Apache
```
Comment (obrienben) [inline docs/manual/owb-to-pywb-deploy.rst:1]: Should this be "with" instead of "and with"?

**Comment by ato** [comment]
The only question that occurred to me was:

How do I run the stuff in sample-deploy? I can probably figure out how to install docker-compose but it might be helpful to include the command(s) I need to run run the docker-compose yaml files.

File: docs/manual/outbackcdx.rst (hunk 4 above)
Comment (ato) [reply docs/manual/outbackcdx.rst:31]: There's no need to reindex for the same level of functionality. The only reasons I can think of to reindex are:
- you have old CDX9 indexes without the compressed length field and access the WARC files over HTTP (the length field enables a correct range request for a single warc record)
- you want to index POST requests (I've never tested this myself but there's some discussion about it in #585)

File: docs/manual/migrating-cdx.rst [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+CDXJ Conversion
+---------------
+
+The native format used by pywb is the :ref:`cdxj-index` with SURT-ordering, which uses JSON to encode the fields, allowing for more flexibility by storing most of the index in a JSON, allowing support for optional fields as needed.
+
+If your CDX are not SURT-ordered, 11 or 9 field CDX, or if there is a mix, pywb also offers a conversion utility which will convert all CDX to the pywb native CDXJ: ::
+
+  wb-manager cdx-convert <dir-of-cdx-files>
+
+The converter will read the CDX files and create a corresponding .cdxj file for every cdx file. Since the conversion happens on the .cdx itself, it does not require reindexing the source WARC/ARC files and can happen fairly quickly.  These files are guaranteed should work with pywb.
```
[the remark this answers is not in the document]
Comment (ikreymer) [reply docs/manual/migrating-cdx.rst:31]: Cleared it up a bit! The format is guaranteed to be correct, but of course other things could go wrong for it not not 'work'!

File: docs/manual/owb-to-pywb-config.rst [hunk 7]
```diff
@@ -0,0 +1,308 @@
...
+
+.. code:: yaml
+
+    collections:
+        wayback:
+            index_paths: ...
+            archive_paths: /archive/warc-paths.txt
+
+
+The path index is a tab-delimited text file for mapping WARC filenames to full paths, eg:
```
Comment (ikreymer) [inline docs/manual/owb-to-pywb-config.rst:225]:
```suggestion
The path index is a tab-delimited text file for mapping WARC filenames to full file paths or URLs, eg:
```

Comment (ikreymer) [reply docs/manual/owb-to-pywb-config.rst:225]: @ldko good point, updating the text and example

File: docs/manual/owb-to-pywb-config.rst [hunk 8]
```diff
@@ -0,0 +1,308 @@
...
+
+.. code::
+
+    example.warc.gz<tab>/some/path/to/example.warc.gz
+    another.warc.gz<tab>/some-other/path/another.warc.gz
```
Comment (ikreymer) [inline docs/manual/owb-to-pywb-config.rst:230]:
```suggestion
    another.warc.gz<tab>/some-other/path/another.warc.gz
    remote.warc.gz<tab>http://warcstore.example.com/serve/remote.warc.gz
```

File: docs/manual/owb-to-pywb-deploy.rst (hunk 5 above)
Comment (ikreymer) [inline docs/manual/owb-to-pywb-deploy.rst:1]:
```suggestion
Deploying pywb: Collection Paths and routing with Nginx/Apache
```

Comment (ikreymer) [reply docs/manual/owb-to-pywb-deploy.rst:1]: Thanks!

**Comment by ikreymer** [comment]
@ldko @ato Updated two pages:
- OutbackCDX info now mentions that regenerating CDX is optional: https://pywb.readthedocs.io/en/docs/manual/outbackcdx.html

- Added more info on how to run Docker Compose examples:
https://pywb.readthedocs.io/en/docs/manual/owb-to-pywb-deploy.html#working-docker-compose-examples

Let me know if this is what you had in mind (and many thanks again for the typo fixes @ldko!)

**Review by ldko — APPROVED** [review]
Thanks for addressing my comments and question, Ilya. This looks good to me!

**Pull request merged**